### ISSUE TITLE
bosh-lite VM fails to start due to Virtualization issue.

### ISSUE THREAD

**Issue by suman007** [cloudfoundry/bosh-lite#359]
Hello ,

I am facing a typical problem while running the bosh-lite VM using vagrant on virtual box.

The error says:

Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'cloudfoundry/bosh-lite' is up to date...
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "1de0a922-b0aa-47fd-a49e-0ff1477a538d", "--type", "headless"]

Stderr: VBoxManage: error: VT-x is not available (VERR_VMX_NO_VMX)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole

I am using a CentOS 7 base machine where I am wrunning vagrant (v 1.8.1) & virtual box (5.0.16 r)

VT-x is already enabled in my host machine.

Can someone help here.

Thanks,
Suman

**Comment by cppforlife** [comment]
Try rebooting. How are you verifying that vt-x is enabled?

**Comment by suman007** [comment]
Hi,

I have rebooted the machine many times. For my host machine, I have verified in virtualization setup  VT-x is enabled their.
I came across a similar post in another thread. It seems we can't run vagrant + VirtualBox within a VirtualBox virtual machine. It is still not supported for oracle virtualbox, but it works now for vmware player.

Here is the link.
http://stackoverflow.com/questions/24620599/error-vt-x-not-available-for-vagrant-machine-inside-virtualbox

Here , where I am at fix  :( .
